clients/nutclient.cpp

```diff
@@ -753,6 +768,57 @@ int TcpClient::deviceGetNumLogins(const std::string& dev)throw(NutException)
 	return atoi(num.c_str());
 }
 
+AsyncResult TcpClient::getAsyncResult(const AsyncID& id)throw(NutException)
+{
+	if (id.empty())
+	{
+		return AsyncResult::SUCCESS;
+	}
+
+	std::string result = sendQuery("GET TRACKING " + id);
+	detectError(result);
+
+	if (result == "PENDING")
+	{
+		return AsyncResult::PENDING;
+	}
+	else if (result == "SUCCESS")
+	{
+		return AsyncResult::SUCCESS;
+	}
+	else if (result == "ERR FAILED")
```

Comment:
As per https://github.com/networkupstools/nut/blob/master/docs/net-protocol.txt#L209
Result can also be:
* ERR UNKNOWN          (command execution failed with unknown error)
* ERR INVALID-ARGUMENT (command execution failed due to missing or invalid argument)
beside from the ERR FAILED (command execution failed)